src/components/ResumeForm.tsx

```diff
@@ -0,0 +1,47 @@
+import { uploadResume } from "@/firebase/storage";
+import { useState } from "react";
+
+function ResumeForm() {
+  type changeEvent = React.ChangeEvent<HTMLInputElement> | undefined
+  type submitEvent = React.FormEvent<HTMLFormElement>
+
+  const [files, setFiles] = useState<Array<File>>()
+
+  function submitHandler(event: submitEvent) {
+    event.preventDefault();
+    if (files) {
+      for (const file of files) {
+        uploadResume(file, "uid", "jobid", "description")
```

Comment:
is this incomplete?

Reply:
No, i had the adding resume function nested inside the upload resume, i have it seperated now.